src/cheat_gui.cpp

```diff
@@ -203,12 +203,14 @@ static const NWidgetPart _nested_cheat_widgets[] = {
 		NWidget(WWT_STICKYBOX, COLOUR_GREY),
 	EndContainer(),
 	NWidget(WWT_PANEL, COLOUR_GREY, WID_C_PANEL), SetDataTip(0x0, STR_CHEATS_TOOLTIP), EndContainer(),
+	NWidget(WWT_PANEL, COLOUR_GREY),
+		NWidget(WWT_TEXTBTN, COLOUR_GREY, WID_C_NOTE), SetFill(1, 1), SetDataTip(STR_CHEATS_NOTE, STR_NULL),
```

Comment:
Did you mean to use WWT_TEXT or WWT_LABEL?
Or do you plan to add some action when pressing it?

Reply:
hahahaha :D Stop finding bugs in my PRs! :P